packages/next-app/src/app/stableswapsimulator/(components)/DepthCost.tsx

```diff
@@ -117,9 +117,9 @@ export function DepthCost() {
 
   const props = {
     data: data,
-    title: "Depth cost (-/+ 2% of price impact)",
+    title: "Depth cost (-/+ 2% of price change)",
```

Comment:
We were getting the concept wrong? ;-;

Reply:
Yes, were wrong. The depth cost calculation is based on spot price. So, it is how much you can add to a token to move the price to -/+ 2%. The price impact is effective price / spot price.